Lib/test/test_os.py

```diff
@@ -472,7 +472,7 @@ def test_access_denied(self):
         # force CreateFile to fail with ERROR_ACCESS_DENIED.
         DETACHED_PROCESS = 8
         subprocess.check_call(
-            ['icacls.exe', fname, '/deny', 'Users:(S)'],
+            ['icacls.exe', fname, '/deny', '*S-1-5-32-545:(S)'],
```

Comment:
Please add a comment mentionning the bpo and explaining what is this identifier, like:

```
# bpo-30584: Use security identifier *S-1-5-32-545 instead of localized "Users"
# to not depend on the locale
```